src/components/organisms/RolesTabs/RolesTabs.tsx

```diff
@@ -81,6 +81,9 @@ const RolesTabs: FC = () => {
     return <div />
   }
 
+  const filteredMainUsers = filter(existingRoles, { is_root: true })
```

Comment:
**[non-blocker]**

Naming is a bit confusing here. We are dealing with roles not users. Right now it's not much of a problem, but could be in the future, when we have added the pages for manipulating users. Will make it slightly harder to find places in the codebase where we are dealing with users